### ISSUE TITLE
Web Tracing Framework no longer Working in Angular

### ISSUE THREAD

**Issue by mhevery** [angular/angular#12509]
**I'm submitting a ...**  (check one with "x")

```
[X] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
1. Navigate to a angular 2 application such as: https://material2-app.firebaseapp.com/
2. Turn on [WTF](http://google.github.io/tracing-framework/)
3. notice that the application goes into a constant refresh loop (i.e. app renders then refreshes etc)

**Expected behavior**
Should come up with WTF UI.

**Minimal reproduction of the problem with instructions**
1. Navigate to a angular 2 application such as: https://material2-app.firebaseapp.com/
2. Turn on [WTF](http://google.github.io/tracing-framework/)
3. notice that the application goes into a constant refresh loop (i.e. app renders then refreshes etc)

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

- **Angular version:** 2.0.X
  
  
- **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
  
  
- **Language:** [all | TypeScript X.X | ES6/7 | ES5]
- **Node (for AoT issues):** `node --version` =

**Comment by alxhub** [comment]
At this point, this is no longer a regression so much as a missing feature, so I'm converting this issue into a feature request.

**Comment by JoostK** [comment]
This was closed by #33949 and WTF seems to have gone unsupported (and modern Web Inspector tooling offers similar functionality)
